debug/rtl.html

```diff
@@ -30,7 +30,8 @@
 <script src='/dist/mapbox-gl-dev.js'></script>
 <script src='/debug/access_token_generated.js'></script>
 <script>
-var pluginURL = 'https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-rtl-text/v0.2.0/mapbox-gl-rtl-text.js';
```

Comment:
nit: Add commentt o indicate what is happening and expected if you plan to commit this change